src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -951,12 +941,18 @@ def generate_PCR_7_default_config(self, filename):
       raise RuntimeError('ConfigError:There is an error in reading path:%s \n\n'
                          % sas_uut_claimed_ppa_boundary_file_path)
 
+    # Shrink the user claimed ppa boundary by approximately 1 kilometer.
+    user_claimed_ppa_contour_shapely = utils.ToShapely(
```

Comment:
I suggest calling this `overlapping_ppa_contour_shapely` since this won't actually be used as the user-claimed PPA contour in this test case.

Reply:
changed as suggested.